.circleci/config.yml

```
@@ -1256,17 +1318,17 @@ jobs:
             equal: [<< parameters.binary_type >>, "solcjs"]
           steps:
             - run:
-                name: External << parameters.project >> tests (solcjs)
+                name: External << parameters.project >> tests (<< parameters.binary_type >>)
                 command: |
-                  test/externalTests/<< parameters.project >>.sh solcjs /tmp/workspace/soljson.js
+                  test/external_tests.py test --solc-binary-type << parameters.binary_type >> --solc-binary-path /tmp/workspace/soljson.js --run << parameters.project >>
```

I'd quote shell values that come from variables as a general rule, even if currently it does not seem necessary:
```suggestion
                  test/external_tests.py test --solc-binary-type "<< parameters.binary_type >>" --solc-binary-path /tmp/workspace/soljson.js --run "<< parameters.project >>"
```

It also makes it look a bit less like you're doing input/output redirection :)